### PR TITLE
[NCL-6755] Create first unit test for Project List

### PR DESCRIPTION
**Pull request by DnsZhou**

### CODE REVIEW

File: src/components/ProjectsList/__tests__/ProjectsList.test.tsx [hunk 1]
```diff
@@ -1,13 +1,33 @@
...
-test('renders ProjectsList', () => {
-  render(
-    <></>
-    // <MemoryRouter>
-    //   <ProjectsList />
-    // </MemoryRouter>
-  );
+import { render, screen } from '@testing-library/react';
+
+describe('display project list page', () => {
```
Comment (matedo1) [inline src/components/ProjectsList/__tests__/ProjectsList.test.tsx:5]: I think `'ProjectList component'` description is more appropriate, at least `page` should not be there as it's just component in our terminology

File: src/components/ProjectsList/__tests__/ProjectsList.test.tsx [hunk 2]
```diff
@@ -1,13 +1,33 @@
...
+  async function loadMocks() {
+    mockProjectsRequest = await import('./data/mock-projects-request.json');
+    mockProjects = mockProjectsRequest.content;
+  }
+
+  beforeEach(async () => {
+    await loadMocks();
+  });
+
+  afterEach(async () => {});
```
Comment (matedo1) [inline src/components/ProjectsList/__tests__/ProjectsList.test.tsx:18]: is this neccessary if it's empty?

File: src/components/ProjectsList/__tests__/ProjectsList.test.tsx [hunk 3]
```diff
@@ -1,13 +1,33 @@
...
+
+  test('renders ProjectList to have the right data and', () => {
```
Comment (matedo1) [inline src/components/ProjectsList/__tests__/ProjectsList.test.tsx:20]: is description valid? `and` seems to be added just by mistake?

**Review by matedo1 — CHANGES_REQUESTED** [review]

**Comment by DnsZhou** [comment]
Hi @matedo1 , all issues were fixed accordingly, also the snapshot test was integrated as well.

**Review by matedo1 — APPROVED** [review]
Thanks!

**Pull request closed without merging**